package.json

```diff
@@ -15,8 +15,9 @@
     "type": "app",
     "displayName": "Users",
     "route": "/users",
-    "home": "/users?filters=active.Active&sort=Name",
+    "home": "/users",
```

Comment:
Ah, no, this won't do at all. It relates to comment earlier in the file. We really want to specify this in the package file (which leaves open the possibility to configuring it elsewhere later), not hardwire it in code.

Reply:
I am adding this back, but the thing is, this will do nothing to effect the default values of users when running as an embedded module. Those values will still be derived from the manifest. The values expressed in this query string happen to coincide with values from the manifest, so it seemed to me to obfuscate the behavior, but it is true that for a top level module the values expressed here will be honored over the values expressed in the manifest.

Reply:
> This will do nothing to effect the default values of users when running as an embedded module.

That is an interesting and important point that I'd not thought of at all. In general, we are going to want different defaults when running as the top-level module and when running embedded. So I suspect the way to go might be to add a separate `InitialParamsWhenRunningAsEmbeddedModule` setting to the package file -- at which point it might also make sense to replace `home` with `InitialParamsWhenRunningAsTopLevelModule`. (I admit these names might need a bit of work.)

But come to think of it, perhaps the more honest thing is to have the invoking module specify the initial parameters that it wants from the embedded module?

Anyway, for now, let's put this back -- so that the package-file setting still determines what our initial settings are when running as a top-level module -- and we'll separately address how best to provide initial parameters when running embedded.